src/app/core/store/shopping/categories/categories.selectors.ts

```diff
@@ -73,8 +90,20 @@ export const getNavigationCategories = (uniqueId: string) =>
     defaultMemoize(projector, CategoryTreeHelper.equals, isEqual)
   )(getCategoryTree, (tree: CategoryTree): NavigationCategory[] => {
     if (!uniqueId) {
-      return tree.rootIds.map(mapNavigationCategoryFromId.bind(tree));
+      return tree.rootIds.map(id =>
+        mapNavigationCategoryFromId.bind({
```

Comment:
Maybe this isn't your code originally, but I think the `bind` construct is a bit weird. 
Maybe we can make it simpler?

Reply:
Definitely not a blocker, but something we should talk about as a team. Conciseness vs. readability, I guess.

Reply:
@dhhyi Can you help to understand the reason to use a binding instead of a function parameter

Reply:
It is less verbose using bind. But yes, it is also not a commonly known feature. Feel free to change it to a function parameter implementation.